packages/vsx-registry/src/browser/vsx-extension.tsx

```diff
@@ -369,8 +369,8 @@ export class VSXExtensionEditorComponent extends AbstractVSXExtensionComponent {
                             {this.renderNamespaceAccess()}
                             {publisher}
                         </span>
-                        {downloadCount && <span className='fa fa-download download-count' onClick={this.openExtension}>{downloadFormatter.format(downloadCount)}</span>}
-                        {averageRating && <span className='average-rating' onClick={this.openAverageRating}>{this.renderStars()}</span>}
+                        {!!downloadCount && <span className='fa fa-download download-count' onClick={this.openExtension}>{downloadFormatter.format(downloadCount)}</span>}
+                        {averageRating !== undefined && <span className='average-rating' onClick={this.openAverageRating}>{this.renderStars()}</span>}
```

Comment:
It does not work, empty stars are still invisible.
<img width="428" alt="Screenshot 2020-03-23 at 16 13 40" src="https://user-images.githubusercontent.com/3082655/77331667-57a4fc00-6d21-11ea-82f9-d42c82b9c4c6.png">

I think we should completely remove a guard here and deal with `undefined` value as zero.

Reply:
Actually it is there but without content, could you have a look please at `renderStars` function. Why it returns empty string.

<img width="507" alt="Screenshot 2020-03-23 at 16 15 08" src="https://user-images.githubusercontent.com/3082655/77331823-89b65e00-6d21-11ea-8e25-a4349c7bb1c4.png">

Reply:
![image](https://user-images.githubusercontent.com/23107734/77355753-6a1c3700-6d1b-11ea-90ee-ce2a61d0b246.png)
![image](https://user-images.githubusercontent.com/23107734/77355856-99cb3f00-6d1b-11ea-8e01-d214a4f821ce.png)

@akosyakov  I think the stars are not displaying because `rating` is `null``.

Reply:
I think `renderStars` should fallback to 0 for `null`.

So the meaning is:
- `undefined` means that an extension is built-in and rating should not be displayed
- `null` means that an extension is published as well but rating is 0